app/services/courses/copy_to_provider_service.rb

```diff
@@ -51,13 +51,17 @@ def copy_latest_enrichment_to_course(course, new_course)
     end
 
     def adjusted_applications_open_from_date(course, year_differential)
-      next_cycle = RecruitmentCycle.next_recruitment_cycle
+      return course.applications_open_from if next_cycle.blank?
```

Comment:
```suggestion
      return current_cycle.application_start_date if course.applications_open_from.blank? && next_cycle.blank?
      
      return course.applications_open_from if next_cycle.blank?
```

Maybe something like this?